### ISSUE TITLE
Does ReduceLROnPlateau verbose works?

### ISSUE THREAD

**Issue by hadaev8** [keras-team/keras#8875]
I think "verbose = 1" should write then learning rate reduced, isn't it?
Looks like now it write only if go to min_lr.

Keras from master, tensorflow, win 10.

**Comment by ozabluda** [comment]
Works fine for me. Try redirecting output to a file and see if maybe the ReduceLROnPlateau output gets overwritten by progbar or something else.

**Comment by hadaev8** [comment]
@ozabluda 
You get notification every time then learning rate reduced?

**Comment by ozabluda** [comment]
Yes, the code is very straightforward, not much room for bugs:
https://github.com/keras-team/keras/blob/38231e8a434f1841b2e5027cc4a48501ac517d1e/keras/callbacks.py#L911

Did you try my suggestion in my previous comment?

**Comment by hadaev8** [comment]
@ozabluda 
I get my error, its write in output minimum value, but not current lr.
Strange in my opinion.

**Comment by hadaev8** [comment]
I get it, if think factor is subtracting from lr and its fail to mon in one step.
